develop/smart-contracts/dev-environments/hardhat.md

```diff
@@ -263,7 +263,7 @@ After testing your contract locally, you can deploy it to a live network. This g
     --8<-- 'code/develop/smart-contracts/dev-environments/hardhat/hardhat.config.js:66:66'
     ```
 
-5. Update your Hardhat configuration file with network settings for the Asset Hub network you want to target:
+5. Update your Hardhat configuration file with network settings for the Polkadot Hub network you want to target:
```

Comment:
This sounds a little bit off to me because if we're calling each network "Westend Hub", "Paseo Hub", "Polkadot Hub", then "Polkadot Hub" is a single network. Should we just call it "Hub network"?